### PR TITLE
feat(across): add uma to fe

### PR DESCRIPTION
**Pull request by Gamaranto**
Signed-off-by: Gamaranto <francesco@umaproject.org>

### CODE REVIEW

File: src/utils/constants.ts [hunk 1]
```diff
@@ -212,7 +211,7 @@ export const TOKENS_LIST: Record<ChainId, TokenList> = {
   [ChainId.ARBITRUM]: [
     {
       address: getAddress("0x82af49447d8a07e3bd95bd0d56f35241523fbab1"),
-      name: "Wrapped Ethereum",
+      name: "Wrapped Ether",
```
Comment (mrice32) [inline src/utils/constants.ts:214]: +1

**Review by mrice32 — COMMENTED** [review]
Looks good! One address is wrong I think.

File: src/utils/constants.ts [hunk 2]
```diff
@@ -226,6 +225,14 @@ export const TOKENS_LIST: Record<ChainId, TokenList> = {
       logoURI: usdcLogo,
       bridgePool: getAddress("0x256C8919CE1AB0e33974CF6AA9c71561Ef3017b6"),
     },
+    {
+      address: getAddress("0xd693ec944a85eeca4247ec1c3b130dca9b0c3b22"),
+      name: "UMA Token",
+      symbol: "UMA",
+      decimals: 18,
+      logoURI: umaLogo,
+      bridgePool: getAddress("0x7355Efc63Ae731f584380a9838292c7046c1e433"),
```
[the remark this answers is not in the document]
Comment (daywiss) [reply src/utils/constants.ts:234]: yea @mrice32 you have the right address according to the doc: https://docs.google.com/document/d/1zAFxC3GDIy9Ig8Smu1Nr5Ubi81ROPNnWVuMOLOWjjKU/edit
this should be updated

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**